### ISSUE TITLE
arrayTranspose does not exist

### ISSUE THREAD

**Issue by screendriver** [true-myth/true-myth#180]
When you import a `Maybe` like this

```ts
import { Maybe } from 'true-myth';

let valid = [Maybe.just(2), Maybe.just('three')];
let allJust = Maybe.arrayTranspose(valid);
```

I get the error

```
Property 'arrayTranspose' does not exist on type 'typeof import("/Users/me/something/node_modules/true-myth/maybe")'
```

**Comment by chriskrycho** [comment]
Interesting. I'm not actually sure *why* that isn't showing up, but I will try to dig in and see what's up here later in the week!

**Comment by screendriver** [comment]
Any news on that one? 🙂 At the moment I have multiple Maybes in one function where I want to map over it but only when **all** of them are a `Just`

**Comment by chriskrycho** [comment]
Alas, I’ve been swamped. Might have a chance to poke at it tomorrow, otherwise apt to be a while again yet.

**Comment by chriskrycho** [comment]
@screendriver ha! I figured this out very quickly by checking out v4.1.1 (current latest): we implemented `arrayTranspose`… *but haven’t published a version which has it! So it’s in our docs but not in the world. 😬

**Comment by chriskrycho** [comment]
(Will be resolved by the forthcoming 5.0 release. Also I'll think on how to avoid that problem in the future.)

**Comment by screendriver** [comment]
Ah! He he, ok. Good to know. Thanks 👍
